### ISSUE TITLE
OKOK implementation does not support my Chipsea-BLE scale (implementation available)

### ISSUE THREAD

**Issue by rrooggiieerr** [oliexdev/openScale#930]
**Describe the bug**
The current OKOK implementation does not support my Chipsea-BLE scale. I have however managed to reverse engineer the communication with the scale and have implemented a working version. I'll great a pull request with the updated code

**To Reproduce**
Reproduced with [latest dev version](https://github.com/oliexdev/openScale/releases/tag/travis-dev-build): ???

**Expected behavior**
I expect the weight of the scale to be read by openScale

**Additional context**
I was actually looking for the protocol create a Home Assistant integration for the scale, but then found that the openScale implementation of the OKOK protocol does not support my scale.

**Debug log**
Was not able to get the BLE debug log from my Android phone, however here is some more info of what I managed to find out using other ways.

The scale identifies as `Chipsea-BLE` on Bluetooth

Manufacturer Specific Data index is `0xf0ff`

```
Manufacturer Specific Data:
 98,7 Kg > 0204db03000000000000000050fb19c01c6d
114,4 Kg > 02047804000000000000000050fb19c01c6d
 98,3 Kg > 0204d703000000000000000050fb19c01c6d
```

Basically byte 2 and 3 are the only bytes that change where the most significant byte is byte 3 and the least significant byte is byte 2

There is no impedance transmitted
There is no checksum

**Comment by oliexdev** [comment]
Thanks for your PR I merged it into the master branch.
